libs/zzz/dm/src/dm/character/character.ts

```diff
@@ -188,14 +189,20 @@ export const charactersDetailedJSONData = Object.fromEntries(
       const raw = JSON.parse(
         readHakushinJSON(`character/${id}.json`)
       ) as CharacterRawData
+      // Not all agents have this info, or it is hidden for some reason
+      const fullname =
+        raw.PartnerInfo.FullName === undefined ||
```

Comment:
do you need a bracket around your first logic here?
`(_||_)?_:_`

Reply:
I think I did but prettier removed it
![image](https://github.com/user-attachments/assets/8d6639a2-9e15-4373-a1fa-3dfa42e7e170)